overlord/patch/patch.go

```diff
@@ -21,16 +21,28 @@ package patch
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+	"time"
 
+	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/overlord/snapstate"
```

Comment:
I've reviewed this as part of a follow up PR by mistake. This cannot be here.. please see the review there for details.